src/FieldWarning/Assets/Units/Component/Vision/VisionComponent.cs

```diff
@@ -51,20 +51,21 @@ public class VisionComponent //: IComponentData
         private MatchSession _session {
             get { return Unit.Platoon.Owner.Session; }
         }
-
-        private VisionComponent() { }
-
-        public VisionComponent(GameObject unitGO, UnitDispatcher unit)
+        
+        public void Initialize(UnitDispatcher dispatcher)
         {
-            _gameObject = unitGO;
-            Unit = unit;
+            Unit = dispatcher;
         }
 
         // Alert all nearby enemy units that they may have to show themselves.
         // Only works if they have colliders!
         public void ScanForEnemies()
         {
-            Collider[] hits = Physics.OverlapSphere(_gameObject.transform.position, max_spot_range, LayerMask.NameToLayer("Selectable"), QueryTriggerInteraction.Ignore);
+            Collider[] hits = Physics.OverlapSphere(
+                    gameObject.transform.position,
+                    max_spot_range,
+                    LayerMask.NameToLayer("Selectable"),
+                    QueryTriggerInteraction.Ignore);
```

Comment:
Do you by any chance know how to make VS default to a double indent when breaking up the same line, or do you use another IDE?

Reply:
> Do you by any chance know how to make VS default to a double indent when breaking up the same line, or do you use another IDE?

I'm using VS Code, so I've mostly been trying to keep to the conventions manually. I must have missed this one though.

Reply:
Actually, I just checked in my IDE and it looks like this one is double-indented

Reply:
I prefer double indent as well honestly, it's not in the conventions because I don't know how to make VS format with it. But I don't mind it if you want to use it.

Reply:
It looks like .editorconfig has a property for this, `continuation_indent_size`, but it's not widely supported in IDEs (yet): https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties